File: app/src/main/java/com/example/picassotutorial/ui/java/DialogActivity.java
package com.example.picassotutorial.ui.java;

import android.annotation.SuppressLint;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import com.example.picassotutorial.R;

@SuppressLint("Registered")
public class DialogActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.view_dialog);
        setTitle("Attention!");
    }
}

File: app/src/main/java/com/example/picassotutorial/ui/DialogActivity.kt
package com.example.picassotutorial.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.picassotutorial.R

class DialogActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.view_dialog)
        title = "Attention!"
    }
}